src/components/Timeline.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import './Timeline.css';
+
 import TimelineEvent from './TimelineEvent';
+import timelineData from '../data/timeline.json';
 
 const Timeline = () => {
-  
-  return;
-}
+  return ( 
+    <div className="timeline">
+    {timelineData.events.map((item, i) => (
+      <TimelineEvent key={i} tlEvent={item} />
```

Comment:
The tests in `TimelineEvent.test.js` expect that the props will be `person`, `status` and `timeStamp`.  That's why those tests failed.

The tests in `Timeline.test.js` also expect that it will be sent an array of events through a prop called `events` you can check it out if you open the test files.